exchanges/stream/websocket_connection.go

```diff
@@ -202,13 +211,13 @@ func (w *WebsocketConnection) ReadMessage() Response {
 				// method on WebsocketConnection type has been called and can
 				// be skipped.
 				select {
-				case w.readMessageErrors <- err:
+				case w.parent.ReadMessageErrors <- err:
```

Comment:
This as well, if you're moving towards a `w` holding many `WebsocketConnections`, sharing `ReadMessageErrors` across them will be very jank and confusing to debug errors